CHANGES.rst

```diff
@@ -6,6 +6,8 @@ Changes by Version
 
 - Added Thrift support to ``tcurl.py`` and re-worked the script's arguments.
 - Changed minimum required version of Tornado to 4.2.
+- Added the ability to customize the attributes ``tchannel.testing.vcr`` uses
```

Comment:
Try "Specify which request components to match on with `tchannel.testing.vcr` - e.g. 'header', 'body', etc..."